ios/CodePush/CodePush.m

```diff
@@ -760,4 +760,29 @@ - (void)applicationWillResignActive
     resolve(nil);
 }
 
+#pragma mark - Methods for handling dispatching of download progress events to JS (Private)
```

Comment:
How about `#pragma mark - RCTFrameUpdateObserver Methods (Private)`? We use this convention in other areas.